native/acter/src/api/tasks.rs

```diff
@@ -236,8 +239,8 @@ impl TaskListDraft {
         self
     }
 
-    pub fn color(&mut self, color: Box<Color>) -> &mut Self {
-        self.content.color(Some(Box::into_inner(color)));
+    pub fn color(&mut self, color: Color) -> &mut Self {
```

Comment:
Is this meant for bg color of task content?

Reply:
no, more like a way to highlight or mark tasks or task lists ... same as the comment for pins, we never fully implemented any product flow for this just anticipated it is something easy and quickly to add..